source/dem/velocity_verlet_integrator.cc

```diff
@@ -15,20 +15,17 @@ VelocityVerletIntegrator<dim>::integrate_half_step_location(
   const std::vector<Tensor<1, 3>> &force,
   const std::vector<double> &      MOI)
 {
+  Tensor<1, dim> particle_acceleration;
```

Comment:
Since the `particle_acceleration` is removed for the loop, would it be a good practice to clear it in the loop before updating it at line 30?

Reply:
That would not be an issue because we are using equality afterwards